build.sbt

```diff
@@ -53,7 +58,12 @@ lazy val common = project
     name := "anghammarad-common",
     // publish settings
     releasePublishArtifactsAction := PgpKeys.publishSigned.value,
-    publishTo := sonatypePublishTo.value
+    publishTo := Some(
+      if (isSnapshot.value)
+        Opts.resolver.sonatypeSnapshots
+      else
+       Opts.resolver.sonatypeStaging
+    ),
```

Comment:
Do we need this? I think we can follow [the above conversation](https://github.com/guardian/anghammarad/pull/56/files#r749150301)? Similarly for the change on line 46.

Reply:
good spot - removing in latest commit.